src/QMCWaveFunctions/LCAO/SoaLocalizedBasisSet.cpp

```diff
@@ -665,4 +613,41 @@ template class SoaLocalizedBasisSet<
 template class SoaLocalizedBasisSet<
     SoaAtomicBasisSet<MultiFunctorAdapter<SlaterCombo<QMCTraits::RealType>>, SoaSphericalTensor<QMCTraits::RealType>>,
     QMCTraits::ValueType>;
+
+
+
+  template<class COT, typename ORBT>
+  void SoaLocalizedBasisSet<COT,ORBT>::initializeSpeciesOffsets()
+{
+  const auto& species_names = ions_.getSpeciesSet().speciesName;
+  int num_species = species_names.size();
```

Comment:
Please use `size_t` for size and also add `const` to prevent potential misuse. So `const size_t`